### ISSUE TITLE
Existing note titles exposed when using "Add a quick note" widget

### ISSUE THREAD

**Issue by Jayto1** [streetwriters/notesnook#3496]
### What happened?

When using the widget to "Add a quick note" there is a brief pause on the notes homepage before being thrown in a new note.

### Steps to reproduce the problem

1. Click "Add a quick note" widget.

### Version

2.6.6

### Platform/OS

iOS

### Relevant log output

*No response*

**Comment by thecodrr** [comment]
I am not able to reproduce this. Can you provide a screencast of the issue?

**Comment by Jayto1** [comment]
The outcome varies for me so I created some vector graphics to illustrate the different outcomes (descriptions are above their respective image):

### 1. Creating a new note through the widget (creating a new note through the quick action menu yields the same results)
![image1](https://github.com/streetwriters/notesnook/assets/79473923/b59968b8-dbe6-440e-b66e-d88d69f9fdfe)
### 2a. After completing step 1 and before reaching step 3 the app displays the home screen (same as if you just opened the app normally)
![image2](https://github.com/streetwriters/notesnook/assets/79473923/513a6c2c-2a41-4a1b-a572-f333e8b0e5ba)
### 2b. Same as 2a but with the keyboard loaded
![image3](https://github.com/streetwriters/notesnook/assets/79473923/2be7a2c3-3a1c-45f7-9088-d9df83a3e3ef)
### 2c. After completing step 1 Notesnook stops here and a new note is never loaded
![image2](https://github.com/streetwriters/notesnook/assets/79473923/a130bf90-1909-4335-a8ec-fe61fa505b67)
### 2d. Same as 2c but with the keyboard loaded
![image3](https://github.com/streetwriters/notesnook/assets/79473923/a4d77975-a8ba-4540-b926-705038323ba7)
### 3. In a new note with the keyboard loaded!
![image4](https://github.com/streetwriters/notesnook/assets/79473923/9c479163-5f17-4de0-aa95-8140435fe7a6)

Basically the home screen always seems to load before the new note, sometimes along with the keyboard. Sometimes the new note just straight up fails to load, sometimes along with the keyboard.

Thanks! :)
